cmd/upgrade/app/v1alpha1/result.go

```diff
@@ -49,9 +46,8 @@ type UpgradeResult struct {
 // UpgradeResultGetOrCreateBuilder helps to get or create UpgradeResult instance
 type UpgradeResultGetOrCreateBuilder struct {
 	*errors.ErrorList
-	SelfName        string
 	SelfNamespace   string
-	SelfUID         types.UID
+	SelfOwner       *metav1.OwnerReference
```

Comment:
Can this be `Owner` ?
Can we write comments for each field?

Reply:
done